docs/how-to-guides/init-and-test-a-flow.md

```diff
@@ -222,7 +222,7 @@ Promptflow CLI provides a way to start an interactive chat session for chat flow
 pf flow test --flow <flow-name> --interactive
 ```
 
-After executing this command, customer can interact with the chat flow in the terminal. Customer can press **Enter** to send the message to chat flow. And customer can quit with **ctrl+Z**.
+After executing this command, customer can interact with the chat flow in the terminal. Customer can press **Enter** to send the message to chat flow. And customer can quit with **ctrl+C**.
```

Comment:
Not sure if we have some image needs to update

Reply:
Update the image with old comment